content/ecosystem/data.toml

```diff
@@ -109,6 +109,7 @@ categories = ["ecs"]
 name = "bgfx-rs"
 source = "crates"
 categories = ["3drendering"]
+archived = true
```

Comment:
Looking for maintainer as of 9 months ago, no real activity on their issue tracker